packages/calling-stateful-client/src/BetaToStableTypes.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// This is for mapping new types from beta, which does not exist in stable sdk.
+// These mappings are necessary to bypass type check of conditional compilation
+// Remove these types when the feature get stabilized in stable sdk
+
+// For example, CallCommon doesn't exist in stable call sdk, so it will be still Call type
+// So we remove CallCommonBeta type in stable
+// In beta, Call | CallCommon = CallCommon because Call is just a super set of CallCommon
+
+import { Call, CallAgent, IncomingCall } from '@azure/communication-calling';
+/* @conditional-compile-remove(teams-identity-support) */
+import {
+  TeamsCall as TeamsCallBeta,
+  CallCommon as CallCommonBeta,
+  CallAgentCommon as CallAgentCommonBeta,
+  TeamsCallAgent as TeamsCallAgentBeta,
+  IncomingCallCommon as IncomingCallCommonBeta
+} from '@azure/communication-calling';
+
+/**
+ * @private
+ */
+export type CallCommon = Call | /* @conditional-compile-remove(teams-identity-support) */ CallCommonBeta;
```

Comment:
Will this become the calling sdk `CallCommon` type when the beta is removed?

Reply:
(likewise with the other 'Commons' in this file?)

Reply:
In beta build
Call | CallCommon = CallCommon (because Call is child class of CallCommon)

In stable build 
It is just simply Call type

Once everything is in stable, this will be still running fine
But we can remove these types and import all "commons" from sdk directly to make it perfect once we get there (just like what comments say at the top of file)